extensions-core/s3-extensions/src/main/java/org/apache/druid/data/input/s3/S3InputSourceConfig.java

```diff
@@ -34,24 +34,44 @@
  */
 public class S3InputSourceConfig
 {
+  @Nullable
+  @JsonProperty
+  private String assumeRoleArn;
+  @Nullable
+  @JsonProperty
+  private String assumeRoleExternalId;
+  @JsonProperty
+  private PasswordProvider accessKeyId;
+  @JsonProperty
+  private PasswordProvider secretAccessKey;
```

Comment:
accessKeyId and SecretAccessKey are also nullable, Can we add annotation these as well.

Reply:
Added.